accumulo/build.sbt

```diff
@@ -7,6 +7,7 @@ libraryDependencies ++= Seq(
     exclude("org.apache.hadoop", "hadoop-client"),
   sparkCore % "provided",
   spire,
+  scaffeine,
```

Comment:
^ bump this question up, as all backends depend on spark, should it be only a `spark` project dep?